src/nextcloud/utilities/nextcloud-utilities

```diff
@@ -18,3 +19,31 @@ wait_for_nextcloud_to_be_configured()
 		printf "done\n"
 	fi
 }
+
+
+
+cronjob_interval()
+{
+	interval="$(snapctl get cron.interval)"
+	if [ -z "$interval" ]; then
+		interval="$DEFAULT_CRONJOB_INTERVAL"
+		set_cronjob_interval $interval
```

Comment:
We need to set the previous one here as well, or the first time the configure hook is run (even for configuring something else) the cron job will be started over.

Reply:
Good catch!